README.md

```diff
@@ -104,6 +104,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [LogRocket](https://blog.logrocket.com/become-a-logrocket-guest-author-7d970eb673f9/) - Up to $350 per piece
   > Technical tutorials with code. Write about anything frontend.
 
+- [Magic](https://magic-fortmatic.typeform.com/to/Wgzsocor) - Up to $300 per piece
+  > Technical tutorials on how to use Magic
```

Comment:
Could you also include a link to the product's website here?

If writers don't know what Formatic/Magic is, it will help them learn more about the product and see if they're a good fit for writing.

Reply:
Hi, I've added the link to Magic's website. Is how I did it okay?